app/models/user.rb

```diff
@@ -2,7 +2,8 @@ class User < ApplicationRecord
   has_many :items
   has_many :orders
 
-  validates_presence_of :password_digest, :role, :active, :name, :address, :city, :state, :zip
+  validates_presence_of :password_digest, :name, :address, :city, :state, :zip
```

Comment:
Cool! Don't need the Active because of boolean testing issues.
We'll add a default value to Role as well for `0`